src/setup/extract-configuration.go

```diff
@@ -105,6 +109,9 @@ func ExtractConfiguration(configFilePath string) Configuration {
 		if parsedConfig.SubExperiments[index].PackageType == "" {
 			parsedConfig.SubExperiments[index].PackageType = defaultPackageType
 		}
+		if parsedConfig.SubExperiments[index].PackagePattern == "" {
```

Comment:
Could you add a default value for handler as well. Right now the default is producer-consumer. According to serverless.com the handler should point to the folder with the go funtion definition. 
https://www.serverless.com/blog/framework-example-golang-lambda-support/
so, assigning it "producer-consumer" should do the trick.

I realise it's not going to change much, since the go deployment is not ready, but I think it should have a default value handled anyway.

Reply:
Should the default value for Runtime in teh Configuration struct be golang in this case? I can add it as well.

Reply:
yes! That makes sense